test/integration/call_list_test.rb

```diff
@@ -82,6 +82,7 @@ class CallListTest < ActionDispatch::IntegrationTest
     end
 
     it 'should time a call out after 8 hours' do
+      sign_out
```

Comment:
Makes absolutely no sense that this test is failing, and yet. Programming is stupid.

Reply:
Failure:
CallListTest::completed calls section#test_0002_should time a call out after 8 hours [/home/ubuntu/dcaf_case_management/test/integration/call_list_test.rb:91]:
Expected false to be truthy.